chain/chain/src/chain.rs

```diff
@@ -2841,13 +2837,13 @@ impl Chain {
 
     /// Gets a block by hash.
     #[inline]
-    pub fn get_block(&mut self, hash: &CryptoHash) -> Result<&Block, Error> {
+    pub fn get_block(&mut self, hash: &CryptoHash) -> Result<Block, Error> {
```

Comment:
so get block still stays with mutable?

Reply:
Yeah, I didn't yet updated all the call-sites to be `&`, that's going to be a separate diff

Reply:
https://github.com/near/nearcore/pull/6889